server/blob/blob-deduplicating/src/main/scala/GC.scala

```diff
@@ -118,8 +118,8 @@ case class Report(iteration: Iteration,
                  )
 
 case class State(blobs: Set[BlobId],
```

Comment:
Do we expect to load every thing in memory at once?

Reply:
For the moment.